Project.toml

```diff
@@ -0,0 +1,13 @@
+name = "BlockNLPModels"
+uuid = "26fb23fc-4a37-41a0-8f1d-1a5a3817d965"
+authors = ["Rishabh Gupta <rishabh@umn.edu>"]
+version = "0.1.0"
+
+[deps]
+Ipopt = "b6b21f68-93f8-5de0-b562-5493be1d77c9"
+JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
+NLPModels = "a4795742-8479-5a88-8948-cc11e1c8c1a6"
+NLPModelsIpopt = "f4238b75-b362-5c4c-b852-0801c9a21d71"
+NLPModelsJuMP = "792afdf1-32c1-5681-94e0-d7bf7a5df49e"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+SolverCore = "ff4d7338-4cf1-434d-91df-b86cb86fb843"
```

Comment:
packages that are not used in `src` can be put under `[extras]`. See https://github.com/MadNLP/MadNLP.jl/blob/master/Project.toml#L27-L34